templates/jobs/jobs.html

```diff
@@ -28,7 +28,7 @@
                             <tr>
                                 <td><a href="{{ job.company.website }}">{{ job.company }}</a></td>
                                 <td><a href="{% url 'jobs:job_details' id=job.id %}">{{ job.title }}</a></td>
-                                <td>{{ job.post_date }}</td>
+                                <td>{{ job.published_date }}</td>
```

Comment:
yes, I think we changed the name of this field, it must have just been missed here